src/librustc_resolve/lib.rs

```diff
@@ -982,7 +977,19 @@ impl<'a> NameBinding<'a> {
     }
 
     fn is_public(&self) -> bool {
```

Comment:
I'd get rid of this function (preferably), or at least rename it to `is_pseudo_public`